src/templates/head-meta.js

```diff
@@ -6,7 +6,7 @@ import { formatMessage } from './utils-common'
  * @this {import('vue/types/vue').Vue}
  * @return {import('vue-meta').MetaInfo}
  */
-export function nuxtI18nHead ({ addDirAttribute = false, addSeoAttributes = false } = {}) {
+export function nuxtI18nHead ({ addDirAttribute = false, addSeoAttributes = false, canonicalQueries = false } = {}) {
```

Comment:
This can default to `undefined` to avoid the type issue and avoid including `boolean` in the type.

Reply:
@rchl I tried defaulting to undefined and it has type issues too. I've never had to deal with a vanilla JS file complain about types before, so I am not sure how to resolve it.

Reply:
I've tested this when making the comment and `canonicalQueries = undefined` here and changing the type to `canonicalQueries?: string[]` worked for me.

I want to avoid the boolean because value `true` makes no sense in that option.